### PR TITLE
Pagecontainer: Remove unnecessary $( ":focus" ) fallback on page change

### LINKED ISSUE #1560: Performance: changePage() searches the entire document for :focus elements
In changePage() I see this:

$( window.document.activeElement || "" ).add( "input:focus, textarea:focus, select:focus" ).blur();

This will cause JQM to search the entire document every time we change pages just to release keyboard focus. Instead, we should probably be using a live("focus") handler and tracking the currently focused element so we don't have to crawl the document.

### PR DESCRIPTION
**Pull request by gabrielschulhof**
Fixes gh-1560

### CODE REVIEW

File: js/widgets/pagecontainer.js [hunk 1]
```diff
@@ -939,7 +939,7 @@ define( [
 				historyDir, pageTitle, isDialog,
 				alreadyThere, newPageTitle,
 				params,	cssTransitionDeferred,
-				beforeTransition;
+				beforeTransition, focusElement;
```
Comment (arschmitz) [inline js/widgets/pagecontainer.js]: Follow style guide for variable declarations

File: js/widgets/pagecontainer.js [hunk 2]
```diff
@@ -1036,21 +1036,18 @@ define( [
 			}
 
 			// Kill the keyboard.
-			// XXX_jblas: We need to stop crawling the entire document to kill focus.
-			//            Instead, we should be tracking focus with a delegate()
-			//            handler so we already have the element in hand at this
-			//            point.
-			// Wrap this in a try/catch block since IE9 throw "Unspecified error" if
+			// Wrap this in a try/catch block since IE9 throws "Unspecified error" if
```
Comment (arschmitz) [inline js/widgets/pagecontainer.js]: Make this a proper support comment

File: js/widgets/popup.js [hunk 3]
```diff
@@ -26,6 +26,17 @@ define( [
 //>>excludeEnd("jqmBuildExclude");
 (function( $, undefined ) {
 
+// Safely handle IE9 fragility
```
Comment (arschmitz) [inline js/widgets/popup.js]: Make this a proper support comment

File: js/widgets/pagecontainer.js (hunk 1 above)
Comment (gabrielschulhof) [inline js/widgets/pagecontainer.js]: I'm not sure what you mean here. Variables that are not being initialized are being listed together at the beginning of the declaration, subsequent lines are indented, the new variable uses complete words. Where does this deviate from the style guide?

**Comment by arschmitz** [comment]
@gabrielschulhof going to put a temp hold on this, because we are considering adding `$.ui.safeActiveElement()` to ui core as well as a form of safe blur ( though the api's will likely be a bit different ).

**Comment by arschmitz** [comment]
@gabrielschulhof if you base this off 1.5-dev we are on up-to-date ui which contains these now

**Comment by arschmitz** [comment]
Can you also land this on 1.5-dev please

**Comment by gabrielschulhof** [comment]
Reopened as https://github.com/jquery/jquery-mobile/pull/8154 against 1.5-dev.

**Pull request closed without merging**